ibm/mas_airgap/roles/ocp_release_mirror/defaults/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+# Get Registry facts from environment variables if they have not been passed to the role
+registry_public_host: "{{ lookup('env', 'REGISTRY_PUBLIC_HOST') }}"
+registry_public_port: "{{ lookup('env', 'REGISTRY_PUBLIC_PORT') }}"
+registry_public_url: "{{ registry_public_host }}:{{ registry_public_port }}"
+
+# Release config
+openshift_release_version: "{{ lookup('env', 'OPENSHIFT_RELEASE_VERSION') }}"
+dir: "{{ lookup('env', 'DIR') }}"
+
+ibm_entitlement_key: "{{ lookup('env', 'IBM_ENTITLEMENT_KEY') }}"
```

Comment:
Similar ... I don't see either ibm_entitlement_key or ibm_auth being used in this role, so think we can remove them from here

Reply:
ibm_entitlement_key is used to create ibm_auth. ibm_auth is used on line 69 of ocp_release_mirror/tasks/main.yml